spring-integration-core/src/main/java/org/springframework/integration/endpoint/AbstractPollingEndpoint.java

```diff
@@ -168,6 +169,9 @@ protected void onInit() {
 					this.taskExecutor = new ErrorHandlingTaskExecutor(this.taskExecutor, this.errorHandler);
 				}
 			}
+			if (this.transactionSynchronizationFactory == null) {
+				this.transactionSynchronizationFactory = new PassthroughTransactionSynchronizationFactory();
+			}
```

Comment:
Well, for that purpose we can document:
"We have `PassthroughTransactionSynchronizationFactory` if you would like to have `TL` value for message in case of custom TX Advice. It is applied by default if TX Advice is exactly `TransactionInterceptor`."